install.sh

```diff
@@ -865,7 +865,7 @@ ${KILLALL_K3S_SH}
 
 if command -v systemctl; then
     systemctl disable ${SYSTEM_NAME}
-    systemctl reset-failed ${SYSTEM_NAME}
+    systemctl reset-failed
```

Comment:
I don't think this is desired either. We don't want to reset **all** the failed units on the host, that would be bad. Did you maybe just want:
```suggestion
    systemctl reset-failed ${SYSTEM_NAME} || true
```